web/controllers/password_controller.ex

```diff
@@ -24,6 +24,10 @@ defmodule Healthlocker.PasswordController do
         conn
         |> put_flash(:error, "Could not send reset email. Please try again later")
         |> redirect(to: password_path(conn, :new))
+      %{role: "clinician"} ->
+        conn
+        |> put_flash(:error, "Could not send reset email. Please try again later")
```

Comment:
Perhaps the error message should be more _helpful_ e.g: please Login to Healthlocker via ePJS (_or the full name of ePJS..._)